esp32_jpeg_enc/main/jpeg_decoder.c

```diff
@@ -0,0 +1,72 @@
+#include <stdio.h>
```

Comment:
The example is for jpeg_decoder and the folder is `esp32_jpeg_enc`?

Reply:
> @Suyash20012000 did you test the code?
> 
> Add a proper description in README

I did not test the code. I was not sure about the 'process' part of the code. Also what are your views on the flow of the code. I mean the flow in which the API's are called and the changes I did to the orignal program. Was it on the right track?